test/commonjs/test-artifact.js

```diff
@@ -2,10 +2,10 @@
 /**
  * Rebuild first
  */
-const {execFileSync} = require('child_process');
+const {execSync} = require('child_process');
 
 console.log('Building CommonJS version...');
-execFileSync('npm', ['run', 'build'], {stdio: 'inherit'});
+execSync('npm run build', {stdio: 'inherit'});
```

Comment:
Can you elaborate on why this change is required?

Reply:
@tinovyatkin `npm` is a batch file on Windows, which can't be executed by `execFile`.